README.md

```diff
@@ -145,15 +131,60 @@ cat <<EOF >lst.xml
   <lib name="sdl"/>
 </tool>
 EOF
-compile the TrackLooper repository if you haven’t
 scram setup lst.xml
 cmsenv
-git cms-checkdeps -a
 scram b -j 12
-cd RecoTracker/LST/test/
-cmsRun LSTAlpakaTester.py
 ```
 
+### Run the LST reconstruction in CMSSW
+A simple test configuration of the LST reconstruction can be run with the command:
+```bash
+cmsRun RecoTracker/LST/test/LSTAlpakaTester.py
```

Comment:
This line gives me some errors also

An exception of category 'ConfigFileReadError' occurred while
   [0] Processing the python configuration file named RecoTracker/LST/test/LSTAlpakaTester.py
Exception Message:
 unknown python problem occurred.
ModuleNotFoundError: No module named 'RecoTracker.MeasurementDet.measurementTrackerEventDefault_cfi'

At:
  /home/users/yagu/TrackLooper_Ntuple/test_PR289/CMSSW_13_0_0_pre4/src/RecoTracker/MeasurementDet/python/MeasurementTrackerEventProducer_cfi.py(3): <module>

Reply:
If `scram` failed, of course you get errors when you try to run the rest...